internal/pkg/template/templates/environment/partials/environment-manager-role.yml

```diff
@@ -313,6 +313,11 @@ EnvironmentManagerRole:
             - 'cloudformation:DeleteStack'
           Resource:
             - !Sub 'arn:${AWS::Partition}:cloudformation:${AWS::Region}:${AWS::AccountId}:stack/${AWS::StackName}/*'
+        - Sid: ListStacks
```

Comment:
Maybe adding this to https://github.com/aws/copilot-cli/blob/b4ccefa2c3d35ccb8988372696ea114db81cc715/internal/pkg/template/templates/environment/partials/environment-manager-role.yml#L106?